src/app/shared/filter-bar/label-filter-bar/label-filter-bar.component.css

```diff
@@ -88,17 +88,19 @@
   flex-direction: row;
   justify-content: flex-start;
   align-items: center;
+  width: calc(100% - 8px);
+  border-radius: 10px;
+  height: 40px;
+  padding: 0px 12px;
+  margin: 8px 4px;
+  box-sizing: border-box;
```

Comment:
Are the height, width and box-sizing properties necessary? 
The component seems to behave the same without these properties.

Reply:
I see your reasoning. 
Makes sense to me.